helm/values.yaml

```diff
@@ -48,6 +48,7 @@ kubernetes:
       runAsGroup: 20001 # remove when using openshift or OKD 4
       seccompProfile: # remove when using Kubernetes prior v1.19, openshift or OKD 4
         type: RuntimeDefault # remove when using Kubernetes prior v1.19, openshift or OKD 4
+    podSecurityContext: {}
```

Comment:
Think you now need to comment that the above is container level sec context or at least make this transparent why there is two

Reply:
Is there an exclusion to havin both?

Reply:
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container

> Security settings that you specify for a Container apply only to the individual Container, and they override settings made at the Pod level when there is overlap. Container settings do not affect the Pod's Volumes.

Reply:
added comment.

> Is there an exclusion to havin both?
i don't understand. there are some setting that can be done in A(container) alone, B(pod) alone and A + B together.